test/run-integration-tests

```diff
@@ -60,12 +60,13 @@ prefix() {
 #
 fetch-ringpop-common() {
     if [ ! -e "$ringpop_common_dir" ]; then
-        run git clone --depth=1 https://github.com/uber/ringpop-common.git "$ringpop_common_dir"
+        run git clone -b ws-partition-healing --depth=1 https://github.com/uber/ringpop-common.git "$ringpop_common_dir"
```

Comment:
make sure the changes regarding the branch selection are removed before merging.